lib/membrane/video_compositor/handler.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Membrane.VideoCompositor.Handler do
+  @moduledoc """
+  Module defining behaviour of handlers.
+
+  Implementing handler allows to provide custom implementation and
+  react to various events, among others by setting a new scene and/or state.
```

Comment:
From this description, someone might misunderstand the `setting state` phrase.

There are a lot of `states` in the video compositor and I'm not sure whether it's clear, that this is only a custom state passed to the callback, that is only set by the user implementing callbacks and isn't used in any part of VC logic at all. Moreover, combining it in one alternative/conjunction with the `scene` specifying VC composition logic, suggests that this state is somehow used by the VC backend. 

I think that some indication that VC doesn't interact with this state at all is needed.